demisto_sdk/commands/content_graph/interface/neo4j/queries/validations.py

```diff
@@ -0,0 +1,119 @@
+from typing import List, Tuple
+
+from neo4j import Transaction, graph
+
+from demisto_sdk.commands.content_graph.common import (
+    Neo4jRelationshipResult,
+    RelationshipType,
+)
+from demisto_sdk.commands.content_graph.interface.neo4j.queries.common import run_query
+
+
+def validate_unknown_content(tx: Transaction, file_paths):
+    file_paths_filter = (
+        f"WHERE content_item_from.path in {str(file_paths)}" if file_paths else ""
+    )
+    query = f"""
+        MATCH (content_item_from)-[r:{RelationshipType.USES}{{mandatorily:true}}]->(n{{not_in_repository:true}})
+        {file_paths_filter}
+        RETURN content_item_from, collect(r) as relationships, collect(n) as nodes_to
+        """
+    return {
+        int(item.get("content_item_from").id): Neo4jRelationshipResult(
+            node_from=item.get("content_item_from"),
+            relationships=item.get("relationships"),
+            nodes_to=item.get("nodes_to"),
+        )
+        for item in run_query(tx, query)
+    }
+
+
+def validate_fromversion(tx: Transaction, file_paths, from_version):
+    file_paths_filter = (
+        f"AND content_item_from.path in {str(file_paths)}" if file_paths else ""
+    )
+    from_version_filter = (
+        "AND toIntegerList(split(content_item_from.fromversion, '.')) >= toIntegerList([6,5,0])"
+        if from_version
+        else ""
+    )
+
+    query = f"""
+        MATCH (content_item_from)-[r:{RelationshipType.USES}{{mandatorily:true}}]->(n)
+        WHERE toIntegerList(split(content_item_from.fromversion, ".")) < toIntegerList(split(n.fromversion, "."))
+        {from_version_filter}
+        {file_paths_filter}
+        RETURN content_item_from, collect(r) as relationships, collect(n) as nodes_to
+    """
+    return {
+        int(item.get("content_item_from").id): Neo4jRelationshipResult(
+            node_from=item.get("content_item_from"),
+            relationships=item.get("relationships"),
+            nodes_to=item.get("nodes_to"),
+        )
+        for item in run_query(tx, query)
+    }
+
+
+def validate_toversion(tx: Transaction, file_paths, to_version):
+    file_paths_filter = (
+        f"AND content_item_from.path in {str(file_paths)}" if file_paths else ""
+    )
+    from_version_filter = (
+        "AND toIntegerList(split(content_item_from.toversion, '.')) >= toIntegerList([6,5,0])"
```

Comment:
Prefer using a constant of the version instead over a hardcoded string inside a query